phpstan.neon

```diff
@@ -6,7 +6,7 @@ parameters:
     checkMissingIterableValueType: false
     paths:
         - '%rootDir%/../../../src/'
-    excludes_analyse:
+    excludePaths:
```

Comment:
You might have missed https://github.com/propelorm/Propel2/pull/1803 PR and you did a lot of redundant work
Might have been easier to build on top of existing work done (which now is merged anyway).

Reply:
Sorry, I didn't see this PR.  Let me know if there is anything we need to do to merge these together.

Reply:
You might want to rebase on top of master now.